### DISCUSSION TITLE
Including Fullscreen Button in Map Exported With to_html()

### DISCUSSION THREAD

**Discussion by diehl** [opengeos/leafmap#993] [Q&A] [answered]
I exported a MapLibre basemap as a test to HTML and am noticing the zoom buttons and fullscreen toggle button available in the map when visualized in my Jupyter notebook are not present in the HTML version of the map. Is there a way to include the fullscreen button in the HTML version?

**Comment by giswqs** [comment]
The exported map should have the same controls as shown in Jupyter Notebook.

See examples at
https://maps.gishub.org/

**Reply by diehl** [reply]
That is not what I'm seeing. Let me get you an example.

**Reply by diehl** [reply]
@giswqs The following code snippet produces a map that has controls while the HTML rendering of the map does not. 
```
import leafmap.maplibregl as leafmap

m = leafmap.Map(
    center=(-98.5795,39.8283), 
    zoom=3,                   
    height='600px')                           
m

m.to_html('basemap.html')
```

**Comment by giswqs** [comment, marked as answer]
Use this. Save the map as HTML before displaying it. Otherwise, you won't see any controls or any layers except the basemap. 

```python
import leafmap.maplibregl as leafmap

m = leafmap.Map(
    center=(-98.5795,39.8283), 
    zoom=3,                   
    height='600px')    
m.to_html('basemap.html')                       
m
```
